### DISCUSSION TITLE
scrum prep Milestone 3 - 5

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#224] [General]
Write your scrums here

**Comment by MrL9000** [comment]
I finished the rest of my tasks (3 of them) and helped the team proof check the formatting of the Milestone #3 Documentation before being turned in.

**Comment by juan-ortiz92** [comment]
I finished my last lecture topic task and some documentation tasks. I also helped revise the document of the third milestone.

**Comment by IvanierB** [comment]
I finished my remaining tasks. (2LTT)

**Comment by Alej24** [comment]
Finished a all of my remaining tasks and a LTT.
